src/core/geometry/qgsabstractgeometry.h

```diff
@@ -409,7 +409,7 @@ struct CORE_EXPORT QgsVertexId
     CurveVertex
   };
 
-  QgsVertexId( int _part = -1, int _ring = -1, int _vertex = -1, VertexType _type = SegmentVertex )
+  explicit QgsVertexId( int _part = -1, int _ring = -1, int _vertex = -1, VertexType _type = SegmentVertex )
```

Comment:
The sip is a bit different. It shows two ctor. One without params and an 
other one with 4 mandatory params. Therefore no explicit keyword needed there.

Envoyé avec AquaMail pour Android
http://www.aqua-mail.com


Le 24 août 2016 9:09:47 PM Nyall Dawson <notifications@github.com> a écrit :

>> @@ -409,7 +409,7 @@ struct CORE_EXPORT QgsVertexId
>>      CurveVertex
>>    };
>>
>> -  QgsVertexId( int _part = -1, int _ring = -1, int _vertex = -1, 
>> VertexType _type = SegmentVertex )
>> +  explicit QgsVertexId( int _part = -1, int _ring = -1, int _vertex = -1, 
>> VertexType _type = SegmentVertex )
>
> Should this be changed in sip too?
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/qgis/QGIS/pull/3434/files/70defce583ed5522239f5c824236785cf548c746#r76119447